### DISCUSSION TITLE
Composer version in SoftwareReport.Common.ps1

### DISCUSSION THREAD

**Discussion by GaborMihaly** [actions/virtual-environments#5310] [Ideas]
Hi All!

Let me share my experience and finding.

I found today this that for `composer --version` command from version 2.3.0 the response will be `Composer 2.3.0 2022-03-30 11:15:36` so the line 161 in images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 file should be
`($(composer --version)) -match "Composer (?<version>\d+\.\d+\.\d+)" | Out-Null`
instead of
`($(composer --version)) -match "Composer version (?<version>\d+\.\d+\.\d+)" | Out-Null`

anyway the software report build will fail with
`Software list with blank version:`
`- Composer`

**Comment by al-cheb** [comment]
@GaborMihaly , Thank you for mentioned about this issue. I will prepare a fix and let you know.

**Reply by al-cheb** [reply]
Fixed: https://github.com/actions/virtual-environments/pull/5313

**Reply by GaborMihaly** [reply]
Great stuff! Thanks for the update and the quick fix!
